internal/registry/client.go

```diff
@@ -0,0 +1,95 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package registry
+
+import (
+	"bytes"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+
+	"github.com/pkg/errors"
+
+	"github.com/elastic/elastic-package/internal/logger"
+)
+
+const (
+	productionURL = "https://epr.elastic.co"
+	stagingURL    = "https://epr-staging.elastic.co"
+	snapshotURL   = "https://epr-snapshot.elastic.co"
+)
+
+var (
+	// Production is a pre-configured production client
+	Production = NewClient(productionURL)
+	// Staging is a pre-configured staging client
+	Staging = NewClient(stagingURL)
+	// Snapshot is a pre-configured snapshot client
+	Snapshot = NewClient(snapshotURL)
+)
+
+// Client is responsible for exporting dashboards from Kibana.
+type Client struct {
+	host string
```

Comment:
nit: s/host/endpoint/g

Reply:
this is the host -- `endpoint` generally refers to a particular API endpoint, in this case what we're capturing is the actual base url of the registry host we're going to hit.

Reply:
what about the protocol? I agree it's an URL, but it's host + proto :)

Reply:
what if I just use `baseURL`?

Reply:
changed the name to `baseURL`